test/blueprints/Do.Test.Blueprints.Service.Test/Configuration/EnvironmentSwitch.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Do.Architecture;
+using System.Net;
+
+namespace Do.Test.Configuration
+{
+    public class EnvironmentSwitch : TestServiceNfr
+    {
+        protected override Application ForgeApplication() =>
+            Forge.New
+                .Service(
+                    business: c => c.Default(),
+                    database: c => c.InMemory(),
+                    greeting: c => c.WelcomePage()
+                        .ForDevelopment(c.WelcomePage("/development"))
+                        .ForProduction(c.WelcomePage("/production"))
+                        .ForStaging(c.WelcomePage("/staging"))
+                );
+    }
+
+    public class DevelopmentEnvironmentSwitch : EnvironmentSwitch
```

Comment:
make these nested under `EnvironmentSwitch` and remove their suffices to have a single class in this nfr file